okhttp/src/main/java/com/squareup/okhttp/MediaType.java

```diff
@@ -31,6 +31,64 @@
   private static final Pattern PARAMETER = Pattern.compile(
       ";\\s*(?:" + TOKEN + "=(?:" + TOKEN + "|" + QUOTED + "))?");
 
+  /**
+   * A {@link MediaType} constant representing the wildcard media type.
+   */
+  public static final MediaType WILDCARD = new MediaType("*/*", "*", "*", null);
+  /**
+   * A {@link MediaType} constant representing the "application/xml" media type.
+   */
+  public static final MediaType APPLICATION_XML =
+                  new MediaType("application/xml", "application", "xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/atom+xml" media type.
+   */
+  public static final MediaType APPLICATION_ATOM_XML =
+                  new MediaType("application/atom+xml", "application", "atom+xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/xhtml+xml" media type.
+   */
+  public static final MediaType APPLICATION_XHTML_XML =
+                  new MediaType("application/xhtml+xml", "application", "xhtml+xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/svg+xml" media type.
+   */
+  public static final MediaType APPLICATION_SVG_XML =
+                  new MediaType("application/svg+xml", "application", "svg+xml", null);
+  /**
+   * A {@link MediaType} constant representing the "application/json" media type.
+   */
+  public static final MediaType APPLICATION_JSON =
+                  new MediaType("application/json", "application", "json", null);
+  /**
+   * A {@link MediaType} constant representing the "application/x-www-form-urlencoded" media type.
+   */
+  public static final MediaType APPLICATION_FORM_URLENCODED =
+                  new MediaType("application/x-www-form-urlencoded",
+                                "application", "x-www-form-urlencoded", null);
+  /**
+   * A {@link MediaType} constant representing the "multipart/form-data" media type.
+   */
+  public static final MediaType MULTIPART_FORM_DATA =
+                  new MediaType("multipart/form-data", "multipart", "form-data", null);
+  /**
+   * A {@link MediaType} constant representing the "application/octet-stream" media type.
+   */
+  public static final MediaType APPLICATION_OCTET_STREAM =
+                  new MediaType("application/octet-stream", "application", "octet-stream", null);
+  /**
+   * A {@link MediaType} constant representing the "text/plain" media type.
+   */
+  public static final MediaType TEXT_PLAIN = new MediaType("text/plain", "text", "plain", null);
+  /**
+   * A {@link MediaType} constant representing the "text/xml" media type.
+   */
+  public static final MediaType TEXT_XML = new MediaType("text/xml", "text", "xml", null);
+  /**
+   * A {@link MediaType} constant representing the "text/html" media type.
+   */
+  public static final MediaType TEXT_HTML = new MediaType("text/html", "text", "html", null);
```

Comment:
Is this appropriate for OkHttp? I assume that if you're reading & writing HTML, you're probably doing it with a webview and not OkHttp.